app/controllers/api/v1/events_controller.rb

```diff
@@ -15,6 +15,19 @@ def create
     end
   end
 
+  def update
+    @event = Event.find(params[:id])
+
+    if @event.update(event)
+      render
+    else
+      render json: {
+        message: 'Validation Failed',
+        errors: @event.errors.full_messages
+      }, status: 422
```

Comment:
and for this status code, use the appropriate symbol: http://futureshock-ed.com/2011/03/04/http-status-code-symbols-for-rails/